db/migrate/20190805170949_change_columns_to_not_null.rb

```diff
@@ -0,0 +1,121 @@
+    change_column_null :host_genomes, :s3_bowtie2_index_path, false
+    change_column_null :host_genomes, :default_background_id, false
+
+    change_column_null :input_files, :name, false
+    change_column_null :input_files, :sample_id, false
+    change_column_null :input_files, :source_type, false
+    change_column_null :input_files, :source, false
+
+    change_column_null :metadata_fields, :display_name, false
+    change_column_null :metadata_fields, :force_options, false
+    change_column_null :metadata_fields, :is_core, false
+    change_column_null :metadata_fields, :is_default, false
+    change_column_null :metadata_fields, :is_required, false
+    change_column_null :metadata_fields, :default_for_new_host_genome, false
+
+    change_column_null :output_states, :output, false
+    change_column_null :output_states, :pipeline_run_id, false
+
+    change_column_null :pipeline_runs, :sample_id, false
+    change_column_null :pipeline_runs, :job_status, false
+    change_column_null :pipeline_runs, :results_finalized, false
+    change_column_null :pipeline_runs, :alignment_config_id, false
+    change_column_null :pipeline_runs, :alert_sent, false
+
+    change_column_null :taxon_counts, :tax_id, false
+    change_column_null :taxon_counts, :tax_level, false
+    change_column_null :taxon_counts, :count, false
+    change_column_null :taxon_counts, :count_type, false
+    change_column_null :taxon_counts, :percent_identity, false
+    change_column_null :taxon_counts, :alignment_length, false
+    change_column_null :taxon_counts, :e_value, false
+    change_column_null :taxon_counts, :pipeline_run_id, false
+
+    change_column_null :users, :email, false
+    change_column_null :users, :name, false
+    change_column_null :users, :encrypted_password, false
+    change_column_null :users, :authentication_token, false
+
+    change_column_null :taxon_lineages, :version_end, false
+    change_column_null :taxon_lineages, :version_start, false
+
+    change_column_null :backgrounds, :name, false
+    change_column_null :backgrounds, :ready, false
+
+    change_column_null :job_stats, :task, false
+    change_column_null :job_stats, :reads_after, false
+    change_column_null :job_stats, :pipeline_run_id, false
+
+    change_column_null :locations, :osm_id, false
+    change_column_null :locations, :locationiq_id, false
+    change_column_null :locations, :lat, false
+    change_column_null :locations, :lng, false
+
+    change_column_null :metadata, :sample_id, false
+    change_column_null :metadata, :metadata_field_id, false
+
+    change_column_null :phylo_trees, :taxid, false
+    change_column_null :phylo_trees, :tax_level, false
+    change_column_null :phylo_trees, :tax_name, false
+    change_column_null :phylo_trees, :user_id, false
+    change_column_null :phylo_trees, :project_id, false
+    change_column_null :phylo_trees, :status, false
+    change_column_null :phylo_trees, :dag_json, false
```

Comment:
This is going to break phylo_tree creation: phylo_trees are always created with dag_json being NULL (https://github.com/chanzuckerberg/idseq-web/blob/9716a80a847ed62a182b3a283146ad81884dac69/app/controllers/phylo_trees_controller.rb#L229). The dag_json column only gets populated once the phylo_tree pipeline is kicked off: https://github.com/chanzuckerberg/idseq-web/blob/9716a80a847ed62a182b3a283146ad81884dac69/app/models/phylo_tree.rb#L255